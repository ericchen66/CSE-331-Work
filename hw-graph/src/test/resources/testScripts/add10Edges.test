# Tests adding ten edges from one node of a graph,
# ensuring the parent and child node are correct.
# Edges travel to different children

# Creates a graph
CreateGraph graph1

# Adds five nodes to the graph
AddNode graph1 node3
AddNode graph1 node2
AddNode graph1 node1
AddNode graph1 node5
AddNode graph1 node4

# Creates ten edges from node1
AddEdge graph1 node1 node2 edge1
AddEdge graph1 node1 node2 edge3
AddEdge graph1 node1 node2 edge2
AddEdge graph1 node1 node3 edge2
AddEdge graph1 node1 node5 edge1
AddEdge graph1 node1 node3 edge1
AddEdge graph1 node1 node5 edge11
AddEdge graph1 node1 node1 edge2
AddEdge graph1 node1 node4 edge1
AddEdge graph1 node1 node4 edge2

# Lists the children of parent node
ListChildren graph1 node1