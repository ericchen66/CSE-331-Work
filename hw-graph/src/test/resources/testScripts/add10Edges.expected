# Tests adding ten edges from one node of a graph,
# ensuring the parent and child node are correct.
# Edges travel to different children

# Creates a graph
created graph graph1

# Adds five nodes to the graph
added node node3 to graph1
added node node2 to graph1
added node node1 to graph1
added node node5 to graph1
added node node4 to graph1

# Creates ten edges from node1
added edge edge1 from node1 to node2 in graph1
added edge edge3 from node1 to node2 in graph1
added edge edge2 from node1 to node2 in graph1
added edge edge2 from node1 to node3 in graph1
added edge edge1 from node1 to node5 in graph1
added edge edge1 from node1 to node3 in graph1
added edge edge11 from node1 to node5 in graph1
added edge edge2 from node1 to node1 in graph1
added edge edge1 from node1 to node4 in graph1
added edge edge2 from node1 to node4 in graph1

# Lists the children of parent node
the children of node1 in graph1 are: node1(edge2) node2(edge1) node2(edge2) node2(edge3) node3(edge1) node3(edge2) node4(edge1) node4(edge2) node5(edge1) node5(edge11)